roles/controlnode-provision/templates/dnsmasq.conf.j2

```diff
@@ -0,0 +1,17 @@
+# Listen on this specific port instead of the standard DNS port
+# (53). Setting this to zero completely disables DNS function,
```

Comment:
Understood

Reply:
DNSMasq just forwards the queries to the resolver defined /etc/resolv.conf, I've changed it to look in /etc/dnsmasq.resolv.conf and It will load the servers from there.